tests/views/test_admin_organisations_page.py

```diff
@@ -69,3 +70,30 @@ def test_post_admin_organisations_new_page(self):
         ).scalar()
         self.assertTrue(created_organisation)
         self.created_objects.append(created_organisation)
+
+    def test_admin_organisation_view_edit_page(self):
+        """
+        Test admin view edit organisation page (/admin/organisation/edit)
+        """
+        url = "/admin/organisation/edit/?id=%d"
+        organisation = self.fx.OrganisationData.pmg
+        self.make_request(
+            url % organisation.id, self.user, follow_redirects=True,
+        )
+        self.assertIn(organisation.name, self.html)
+
+    def test_admin_update_organisation_with_the_same_subscriptions(self):
+        """
+        Test admin update organisation and keep its subscriptions
+        the same (/admin/organisation/edit).
+        """
+        url = "/admin/organisation/edit/?id=%d"
+        organisation = self.fx.OrganisationData.pmg
+        self.create_organisation_data["subscriptions"] = self.fx.CommitteeData.arts.id
+        response = self.make_request(
+            url % organisation.id,
+            self.user,
+            data=self.create_organisation_data,
+            method="POST",
+            follow_redirects=True,
+        )
```

Comment:
are we missing an assertion here or is a request without an exception sufficient?